docs/api/variables/stringArrayFooter.md

```diff
@@ -6,4 +6,4 @@
 
 ## Defined in
 
-[packages/core/src/parsing.ts:41](https://github.com/ai16z/eliza/blob/main/packages/core/src/parsing.ts#L41)
+[packages/core/src/parsing.ts:41](https://github.com/madjin/eliza/blob/main/packages/core/src/parsing.ts#L41)
```

Comment:
All changes up to here now point to your fork instead of the `ai16z` repo - is that intentional?

Reply:
no def not intentional, will fix - thanks!